FHIR3-0-2-MM202001-Dev/PDFA-3-0/XIS-Server/medmij-pdfa-fhir3-0-2-xis-1-4-serve-2-pdfa.xml

```diff
@@ -39,13 +39,19 @@
         <description value="OAuth Token for patient XXX_Baltus"/>
     </variable>   
     <variable>
-        <name value="binary1-url"/>
-        <expression value="Bundle.entry.select(resource as DocumentReference).where(type.coding.where(code ='68688-1')).content[0].attachment.url.substring(7)"/>
+        <name value="pdfa1-url"/>
+        <!-- Pick the absolute url as is or create an absolute url when a relative url is given by combining the base URL given in the Bundle self link with the relative reference.-->
+        <expression value="iif(Bundle.entry.select(resource as DocumentReference).where(type.coding.where(code ='68688-1')).content[0].attachment.url.startsWith('Binary/'),
+            Bundle.link.url.replaceMatches('DocumentReference[/?].*$','') + Bundle.entry.select(resource as DocumentReference).where(type.coding.where(code ='68688-1')).content[0].attachment.url,
+            Bundle.entry.select(resource as DocumentReference).where(type.coding.where(code ='68688-1')).content[0].attachment.url)"/>
```

Comment:
Voor de leesbaarheid is het wel lekker om 
`Bundle.entry.select(resource as DocumentReference).where(type.coding.where(code ='68688-1')).content[0].attachment.url`

in een losse variabele te definiëren.